src/api.js

```diff
@@ -0,0 +1,17 @@
+import 'whatwg-fetch';
+const rootUrl = 'http://api.openweathermap.org/data/2.5/weather?q=';
+const APIKey = '&APPID=81ebd306a97bf6e5342257562b958514';
+
+module.exports = {
+  getDataFor(city) {
+    const url = rootUrl + city + APIKey;
+    return fetch(url)
+      .then(response => {
+	return response.json()
+      });
+  },
+
+  flushPromises() {
```

Comment:
Are you using this function for the tests only? Maybe you can move that code to the test file then?

Reply:
Correct! Ok, it was there then moved it into api.js...